DEP Tasks/Trainers Task/JSCore/Task 5/Array-Helpers.js

```diff
@@ -2,53 +2,30 @@
 // https://www.codewars.com/kata/array-helpers
 
 Array.prototype.square = function () {
-  var arr = [];
-  for (var i = 0; i < this.length; i++) {
-    arr.push(this[i] * this[i]);
-  }
-  return arr;
+  return this.map(function (n) {
+    return n * n;
+  });
 }
-
 Array.prototype.cube = function () {
-  var arr = [];
-  for (var i = 0; i < this.length; i++) {
-    arr[i] = this[i] * this[i] * this[i];
-  }
-  return arr;
+  return this.map(function (n) {
+    return n * n * n;
+  });
 }
-
 Array.prototype.sum = function () {
-  var sum = 0;
-  for (var i = 0; i < this.length; i++) {
-    sum += this[i];
-  }
-  return sum;
+  return this.reduce(function (a, b) {
+    return a + b;
+  }, 0);
 }
-
 Array.prototype.average = function () {
-  var sum = 0;
-  for (var i = 0; i < this.length; i++) {
-    sum += this[i];
-  }
-  return sum / this.length;
+  return this.sum() / this.length;
 }
-
 Array.prototype.even = function () {
-  var arr = [];
-  for (var i = 0; i < this.length; i++) {
-    if (this[i] % 2 == 0) {
-      arr.push(this[i]);
-    }
-  }
-  return arr;
+  return this.filter(function (item) {
+    return 0 == item % 2;
```

Comment:
Use strict `===` equality while comparing values. Also re-arrange the expression to `item % 2 === 0`.